Content.Server/Fluids/Components/SprayComponent.cs

```diff
@@ -38,4 +41,17 @@ public sealed partial class SprayComponent : Component
     [ViewVariables(VVAccess.ReadWrite), DataField(required: true)]
     [Access(typeof(SpraySystem), Other = AccessPermissions.ReadExecute)] // FIXME Friends
     public SoundSpecifier SpraySound { get; private set; } = default!;
+
+    /// <remarks>
+    /// DeltaV - ATMOS Extinguisher Nozzle
+    /// </remarks>
+    [ViewVariables(VVAccess.ReadWrite), DataField]
+    public SlotFlags TargetSlot;
+
+    [ViewVariables(VVAccess.ReadWrite), DataField]
+    public EntityWhitelist? ProviderWhitelist;
+
+    [ViewVariables(VVAccess.ReadWrite), DataField]
+    public bool ExternalContainer = false;
```

Comment:
remove ViewVariables from these its implicit for datafield
remove `= false` its the default